### PR TITLE
[v0.1.dev1] Merging ej-test with changes to compute_units_to_build()

### PR DESCRIPTION
**Pull request by alephcero**
Related to issue #30 
- Replaces the `to_frame()` method for `local.copy()` for the `parcels` table
- Establishes two different criteria for the creation of the `feasibility` DataFrame if there are several proposals to keep in the SqFtProForma.   
- Redesigns the `compute_units_to_build()` function, replacing the aggregate number of agents to allocate with a DataFrameWrapper with the agents information  and the number of units with the name of the type of units for those agents. This function can now return an integer or a data frame for the units to build, as wells as taking the target vacancy as float or as an array.

### CODE REVIEW

File: urbansim_parcels/utils.py [hunk 1]
```diff
@@ -770,7 +781,7 @@ def run_feasibility(parcels, parcel_price_callback,
     pf = (sqftproforma.SqFtProForma.from_yaml(str_or_buffer=cfg)
           if cfg else sqftproforma.SqFtProForma.from_defaults())
     sites = (pl.remove_pipelined_sites(parcels) if pipeline
-             else parcels.to_frame())
+             else parcels.local)#to_frame())
```
Comment (smmaurer) [inline urbansim_parcels/utils.py]: Typo?

File: setup.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 setup(
     name='urbansim_parcels',
-    version='0.1dev',
+    version='0.1dev1',
```
Comment (smmaurer) [inline setup.py]: Should have another dot to match `__init__.py` -- 0.1.dev1

**Review by smmaurer — CHANGES_REQUESTED** [review]
Looks good!

It's often helpful to distinguish breaking and non-breaking changes for the release notes.

For example, it looks like the new `compute_units_to_build()` will break downstream code that used the previous API. (Although perhaps it's only used locally, by `run_developer()` in the same file?) Whereas the feasibility change adds functionality without breaking anything, and I'm guessing the `parcels.local` change is a bug fix.

**Comment by smmaurer** [comment]
The Travis builds are failing for reasons unrelated to this PR; i'll see if i can fix it.

File: urbansim_parcels/utils.py (hunk 1 above)
Comment (alephcero) [reply urbansim_parcels/utils.py]: Solved in https://github.com/urbansim/urbansim_parcels/pull/32/commits/2a23d734298cb1650a80beb45d41acb88d8371d1

File: setup.py (hunk 2 above)
Comment (alephcero) [reply setup.py]: Solved in https://github.com/urbansim/urbansim_parcels/pull/32/commits/2a23d734298cb1650a80beb45d41acb88d8371d1

**Review by alephcero — COMMENTED** [review]
Ok, Sam! What do you think it's the best way to explicitly say this are breaking changes? In version number or with a warning in CHANGELOG.md?

**Comment by smmaurer** [comment]
Thanks @alephcero, and sorry for the slow reply! 

About the breaking changes, i just meant to mention it in the change log. (And it's a good reason to make sure we're incrementing the dev version number.) It looks like this package hasn't even been declared a v0.1 yet, so i think the most important thing is just to make clear in the change log and PR writeups how the API is changing..

**Review by smmaurer — APPROVED** [review]

**Pull request merged**